base/data/src/main/java/org/openscience/cdk/Atom.java

```diff
@@ -536,6 +536,10 @@ private static boolean parseAtomSymbol(IAtom atom, String str) {
             atom.setAtomicNumber(0);
             atom.setSymbol("R");
             return true;
+        } else if ("*".equals(str)) {
```

Comment:
This isn't an API change (API change = signature change this would be functionality change)... it actually used to be possible: https://github.com/cdk/cdk/commit/c30256c6f52451e0472555220dad57e55ef7015b.
However I realised this isn't the correct fix - I'll send that though now.